ui/app/components/clients/page/sync.hbs

```diff
@@ -2,7 +2,7 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: BUSL-1.1
 ~}}
-{{#if @isSecretsSyncActivated}}
+{{#if @showSecretsSync}}
```

Comment:
I'm not sure we want this conditional here 🤔 `@showSecretsSync` already wraps the tabs so we're hiding this route in cases where secret syncing is not possible. So I think this should be updated to just check for activation so it can render the `No data is available because Secrets Sync has not been activated.` empty state below. 

No more granular checks are needed because the template renders a separate empty state for 0 sync clients.

Reply:
ah yes, this is partially what i meant by "add more granular checks" in my TODO comment above, but i can see that was not obvious at all. 😂

appreciate you noticing / clarifying!